packages/elements/src/components-v3/InputGroup/index.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import { ElInputGroup } from './__styles__'
+import { Input } from '../Input'
+import { Icon, IconNames } from '../Icon'
+import { Label } from '../Label'
+
+export interface IInputGroup extends React.InputHTMLAttributes<HTMLInputElement> {
+  icon?: IconNames
+  label?: string
+  className?: string
+  id?: string
+}
+
+const generateRandomId = (): string => {
+  try {
+    return Math.random().toString(36).substring(7)
+  } catch (e) {
+    return ''
+  }
+}
+
+export const InputGroup: React.FC<IInputGroup> = ({
+  icon,
+  label,
+  className,
+  id = generateRandomId(),
```

Comment:
Can we memoize generateRandomId so the id doesn't change at each render? I think it's much more efficient for React's DOM diffing.